### ISSUE TITLE
Post tags issues

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#1326]
**Please complete the following information** 
Site repository URL: github.com/ONSdigital/sdg-indicators
Open SDG version: not sure when this occured but it's on 1.4.0 and 1.5.0

**Describe the bug**
Tags are no longer properly styles and in high contrast mode font is white rather than black on yellow so inaccessible

**To Reproduce**
1. Go to https://sdgdata.gov.uk/updates/2019/11/04/accessibility.html
3. Scroll down to bottom of post

**Expected behavior**
Tag should be styled as a tag and in high contrast mode font should be black

**Screenshots**
![image](https://user-images.githubusercontent.com/43404358/132678968-7aac5755-eb5d-4c8e-b725-3c6d97d48508.png)
